src/frontends/onnx/tests/requirements.txt

```diff
@@ -1,4 +1,4 @@
 # ONNX - generate test models
 docopt~=0.6.2
-onnx==1.12.0
+onnx==1.13.0
```

Comment:
Can we update C++ version only for ONNX?

Reply:
Done, the CI is running and will show potential issues, if any.